utils/oscap-xccdf.c

```diff
@@ -139,6 +139,7 @@ static struct oscap_module XCCDF_EVAL = {
 		"INPUT_FILE - XCCDF file or a source data stream file\n\n"
         "Options:\n"
         "   --profile <name>\r\t\t\t\t - The name of Profile to be evaluated.\n"
+	"   --rule <name>\r\t\t\t\t - The name of a single rule to be evaluated.\n"
```

Comment:
@dahaic No, we use tabs for indentation. That's a general rule for all our code written in C. As I mentioned in one of previous comments:

>We follow the Linux Kernel coding style. https://www.kernel.org/doc/html/v4.10/process/coding-style.html Unfortunately, some old code doesn't follow the coding style and uses spaces. In new pull request we use tabs, so that's why there is a mix. We haven't replaced spaces by tabs to avoid merge conflicts and possible problems with downstream patches.

Also, profile is not necessary to specify.